src/app/pages/ArticlePage/Byline/index.tsx

```diff
@@ -37,6 +36,28 @@ const Byline = ({ blocks, children }: PropsWithChildren<Props>) => {
     ['model', 'blocks', 0, 'model', 'blocks', 0, 'model', 'text'],
     jobRoleBlock,
   );
+  const twitterText = pathOr(
```

Comment:
One for discussion but I feel like this data processing should happen outside of the component maybe in the BFF or at least in the ArticlePage component. That would make this component a lot easier to reuse

Reply:
@jroebu14 
I agree this should be done at the BFF level at some point, I can create a separate ticket for it. As for the data processing, the component looks to be only scoped to Article pages so I'm not sure it's going to be used elsewhere in the near future. In the parent ticket, there are actually some [pre-processor rules](https://github.com/bbc/simorgh/pull/10240/files#diff-532661096aacbc298d4296f8aa4ca1e6b885a82640033b8a89157a22dccfd3e2) so maybe we can move it to there? There is a separate ticket (WSTEAM1-139) to go over this component again and implement the component standards, so perhaps we can scope that work to be a part of that? Just so we can get this PR moving along, what do you think?

Reply:
OK cool. That sounds good to me! Thanks